README.md

```diff
@@ -27,7 +27,7 @@ New tools are being added frequently, so check this page again!
 | Cuda                   | [clang-format]        |                  |
 | CSS, Less, Sass        | [Prettier]            |                  |
 | Go                     | [gofmt] or [gofumpt]  |                  |
-| HCL (Hashicorp Config) | [terraform] fmt       |                  |
+| HCL (Hashicorp Config) | [terraform] fmt       | [tfsec]          |
```

Comment:
curious for your thoughts on https://github.com/aspect-build/rules_lint/issues/155#issuecomment-1958577686 - maybe this row shouldn't be "HCL" because in practice all tooling for that "language" is actually specific to one of the dialects.

Reply:
Yeah, tfsec is Terraform specific iiuc.

Reply:
okay I'll rename this row in a separate PR